server/integration_tests/test_data/e2e_triple/whatvirusspeciesarers13317andrs7903146/chart_config.json

```diff
@@ -0,0 +1,257 @@
+{
+  "client": "test_detect-and-fulfill",
+  "config": {
+    "categories": [
+      {
+        "blocks": [
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "answerTableTileSpec": {
+                      "columns": [
+                        {
+                          "header": "->alignmentQuality",
```

Comment:
future, I assume header will be improved in future?

Reply:
yes, header will be improved in the future!